frontend/src/actions/taskActions.js

```diff
@@ -259,8 +263,25 @@ const syncTaskError = error => {
   return { type: SYNC_TASK_ERROR, completed: true, error: error }
 }
 
+/*
+ * Task Report
+ */
+
+const reportTaskRequested = () => {
+  return { type: REPORT_TASK_REQUESTED, completed: false }
+}
+
+const reportTaskSuccess = () => {
+  return { type: REPORT_TASK_SUCCESS, completed: true }
+}
+
+const reportTaskError = error => {
+  return { type: REPORT_TASK_ERROR, completed: true, error: error }
+}
+
 const createTask = (task, history) => {
-  validToken()
+  // eslint-disable-next-line no-console
```

Comment:
@longj724 why you removed the token function and added debug, I think you were testing and forgot these lines here.

Reply:
Yes that must have been a mistake - will fix this and everything else.